fix: Change to use database environment variables in dev mode (#8951)
Unleash/unleash: website/docs/contributing/backend/overview.md

export TEST_DATABASE_URL=postgres://unleash_user:password@localhost:5432/unleash_test
```

> However, you cal also use your pre-defined database with env vars:

Reviewer: ```suggestion
Alternatively, you can use your pre-defined database with environment variables:
```

```

> export DATABASE_USERNAME=YOUR-DATBASE-USERNAME

Reviewer: ```suggestion
export DATABASE_USERNAME=YOUR-DATABASE-USERNAME
```

export DATABASE_PASSWORD=YOUR-DATABASE-PASSWORD
export DATABASE_HOST=YOUR-DATABASE-HOST
export DATABASE_PORT=YOUR-DATABASE-PORT
export DATABASE_NAME=YOUR-DATABASE-NAME
export DATABASE_SCHEMA=YOUR-DATABASE-SCHEMA
export DATABASE_APPLICATION_NAME=YOUR-DATABASE-APPLICATION_NAME
export DATABASE_DISABLE_MIGRATION=YOUR-DATABASE-DISABLE_MIGRATION
```

> You can check what each option means on [Database configuration](../../using-unleash/deploy/configuring-unleash.mdx#database-configuration)

Reviewer: ```suggestion
You can find more details about each environment variable in [Configure Unleash](../../using-unleash/deploy/configuring-unleash.mdx#database-configuration).
```
